homework10/zanovo10/ViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ViewController.swift
+//  zanovo10
+//
+//  Created by qeqwe on 20.11.2023.
+//
+
+import UIKit
+
+final class ViewController: UIViewController {
+    
+    private lazy var openCollectionButton: UIButton = {
+        let buttons = UIButton()
+        buttons.setTitle("Collection", for: .normal)
+        buttons.frame = CGRect(x: 130, y:150, width: 150, height: 80)
+        buttons.backgroundColor = .black
+        buttons.addTarget(self, action: #selector(buttonCollectionViewDidTapped), for: .touchUpInside)
+        return buttons
+    }()
+    
+    private lazy var openTableButton: UIButton = {
+        let buttons = UIButton()
+        buttons.setTitle("Table", for: .normal)
+        buttons.frame = CGRect(x: 130, y: 300, width: 150, height: 80)
+        buttons.backgroundColor = .black
+        buttons.addTarget(self, action: #selector(buttonTableViewDidTaped), for: .touchUpInside)
+        return buttons
+    }()
+    
+    private lazy var openScrollButton: UIButton = {
+        let buttons = UIButton()
+        buttons.setTitle("Scroll", for: .normal)
+        buttons.frame = CGRect(x: 130, y: 450, width: 150, height: 80)
+        buttons.backgroundColor = .black
+        buttons.addTarget(self, action: #selector(buttonScrollViewDidTapped), for: .touchUpInside)
+        return buttons
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        view.addSubview(openCollectionButton)
+        view.addSubview(openTableButton)
+        view.addSubview(openScrollButton)
```

Comment:
```suggestion
        view.backgroundColor = .white
        view.addSubview(openCollectionButton)
        view.addSubview(openTableButton)
        view.addSubview(openScrollButton)
```
вынести в отдельный метод